backbone/src/main/java/org/researchstack/backbone/model/survey/SurveyItemType.java

```diff
@@ -57,6 +57,8 @@
     ACCOUNT_EMAIL_VERIFICATION  ("emailVerification"   ),   // EmailVerificationStep
     @SerializedName("externalID")
     ACCOUNT_EXTERNAL_ID         ("externalID"          ),   // ExternalIDStep
```

Comment:
ACCOUNT_EXTERNAL_ID, which currently produces a non-implemented step, is intended to support this feature. We should use this. 

@dephillipsmichael, does that sound right? 

See:

https://github.com/Sage-Bionetworks/BridgeAppSDK/blob/6a135f98a2e7c4020f278a76fcdf97ff2f5fc777/BridgeAppSDKSample/StudyOverviewViewController.swift#L63